src/api/emulations/nfs.js

```diff
@@ -199,6 +192,7 @@ class NFS {
   /**
   * Helper function to create and save file to the network
   * @param {String|Buffer} content - file contents
+  * @param {String|Buffer} userMetadata - optional user metadata
```

Comment:
This param is not part of the function's signature

Reply:
@bochaco Good catch. Thank you.